cmd/qat_plugin/qat_plugin_test.go

```diff
@@ -54,10 +54,6 @@ func TestScanPrivate(t *testing.T) {
 		maxDevNum       int
 		expectedDevNum  int
 	}{
-		{
-			name:        "Return error for uninitialized device plugin",
-			expectedErr: true,
-		},
```

Comment:
is it invalid case?

Reply:
Yes, the device plugin would return no error even if it is not supplied any flag and hence this would be an invalid case.

Reply:
I'd rather update the case to
```
{
    name: "Return no error for uninitialized device",
},
```
instead of removing. I suspect removing the case will lead to some code being not covered by the unit tests.

Reply:
Okay! Thanks for the input. Will update

Reply:
Done!